packages/runtime/src/plugins/dom/index.ts

```diff
@@ -0,0 +1,35 @@
+import { FederationRuntimePlugin } from '../../type/plugin';
+import { loadEntryDom } from './loadEntry';
+
+export function domPlugin(): FederationRuntimePlugin {
+  return {
+    name: 'dom-plugin',
```

Comment:
looks like the `domPlugin` not be used anymore ?

Reply:
yes, it was better to use a fallback that is not the plugin so we handle the case where we want to use defaults